sandbox/acme/src/app/Controllers/AcmeController.php

```diff
@@ -7,7 +7,7 @@
 
 class AcmeController extends BaseController
 {
-    public function testAction()
```

Comment:
If you will be digging in the main module controllers, I'm not using it. Would you mind to create a pull request to have the `getActionSuffix()`